components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -341,6 +351,48 @@ To select the environment:
 
 {{< figure src="/images/automate/update-node-environment.png" alt="Update Node Environment">}}
 
+##### Run List
+
+You can edit, expand or collapse the run list of a node.
+
+{{< figure src="/images/automate/run-list-of-a-node.png" alt="Run List of a Node">}}
+
+Chef Infra allows you to edit the run list of a node by following the steps below:
+
+1. Select **edit** option. A popup window opens.
+
+2. The left section the window contains the list of environment of the nodes. Select any one environment from the list.
+
+3. The right section of the window will show the current run list of the selected environment.
+
+4. Select the run list you want for the specific environment and select **Create**.
+
+The popup window looks like as shown below:
+
+{{< figure src="/images/automate/edit-run-list.png" alt="Edit Run List">}}
+
+Chef Infra allows you to edit the run list of a node using a shortcut. Select the option **Edit Run List** from the ellipses icon of the specific node in the node list. Selecting the option will open the same popup window which can be used to edit the run list of a node.
+
+The ellipses icon list looks like:
+
+{{< figure src="/images/automate/edit-run-list-ellipses-icon.png" alt="Edit Run List Ellipses Icon">}}
+
+#### Manage Tags
+
+Chef Infra allows you to manage tags of the environment. You can add or remove multiple tags by:
+
+- Selecting the node and add the tags from the text bar below the dropdown menu of environment.
+
+- OR, by simply selecting the **Manage Tags** option in the ellipses icon of a specific node. Selecting the Manage Tags option from the ellipses icon opens a popup window, where you can add multiple tags and click **Update Tags**. The popup window looks like as shown below:
+
+{{< figure src="/images/automate/manage-tags-from-ellipses-icon.png" alt="Manage Tags">}}
+
+#### Reset a Node Key
+
+You can reset the key of a node by selecting the **Reset Key** option from the ellipses icon of a specific node in the node list. Selecting the option opens a warning popup specifying that the current key will no longer be accepted. Select **Reset Key** once again to confirm. The popup looks like:
```

Comment:
I'm confused about this section. The "node key" isn't a concept in Manage. Should it be client key?

Reply:
I have updated the documentation